ghost/i18n/locales/es/portal.json

```diff
@@ -9,168 +9,168 @@
+    "If you've completed all these checks and you're still not receiving emails, you can reach out to get support by contacting {{supportAddress}}.": "Si haz comprobado todo y aun no estás recibiendo los correos electrónicos, comunicarte con soporte técnico contactando a {{supportAddress}}.",
+    "In the event a permanent failure is received when attempting to send a newsletter, emails will be disabled on the account.": "En el caso de que haya un problema permanente al intentar enviar un boletín, los correos electrónicos serán desabilitados en su cuenta.",
+    "In your email client add {{senderEmail}} to your contacts list. This signals to your mail provider that emails sent from this address should be trusted.": "Añade este correo electrónico {{senderEmail}} a tu lista de conactos confiados. Esto le indica a tu provedor de correo electrónico, que puedes recibir correos de esta dirección de correo sin problemas.",
+    "Invalid email address": "Correo electrónico invalido",
+    "Jamie Larson": "Jamie Larson",
+    "jamie@example.com": "jamie@example.com",
     "Less like this": "Menos como esto",
     "Make sure emails aren't accidentally ending up in the Spam or Promotions folders of your inbox. If they are, click on \"Mark as not spam\" and/or \"Move to inbox\".": "Asegúrate de que los correos electrónicos no terminen accidentalmente en las carpetas de correo no deseado o promociones de su bandeja de entrada. Si lo son, haz clic en \"Marcar como no spam\" y/o \"Mover a la bandeja de entrada\".",
     "Manage": "Administrar",
     "Maybe later": "Tal vez más tarde",
     "Memberships unavailable, contact the owner for access.": "Membresía no disponible, contacta al propietario para obtener acceso.",
-    "month": "",
+    "month": "mes",
     "Monthly": "Mensual",
     "More like this": "Más como esto",
     "Name": "Nombre",
     "Need more help? Contact support": "¿Necesitas más ayuda? Contacta a soporte técnico.",
-    "Newsletters can be disabled on your account for two reasons: A previous email was marked as spam, or attempting to send an email resulted in a permanent failure (bounce).": "Los boletines se pueden deshabilitar en tu cuenta por dos razones: Un correo electrónico anterior se marcó como correo no deseado o el intento de enviar un correo electrónico resultó en una falla permanente (bounce).",
+    "Newsletters can be disabled on your account for two reasons: A previous email was marked as spam, or attempting to send an email resulted in a permanent failure (bounce).": "Los boletines se pueden deshabilitar en tu cuenta por dos razones: Un correo electrónico anterior se marcó como correo no deseado o el intento de enviar un correo electrónico resulta en una falla permanente (rebota el correo al remitente).",
```

Comment:
I think that the past tense (resultó ) is probably more correct here?  We're saying that the email bounced in the past, not that the email is bouncing now (because we have given up trying).